neon_intrinsics/advsimd.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Arm Neon Intrinsics Reference
-version: Development version based on 2021Q2
-date-of-issue: 02 July 2021
+version: Development version based on 2021Q3
+date-of-issue: TBD
 # LaTeX specific variables
 landscape: true
 copyright-text: Copyright © 2020-2021, Arm Limited and its affiliates. All rights reserved.
```

Comment:
```suggestion
copyright-text: Copyright: see section \hyperlink{Copyright}{copyright}.
```

I am not actually sure about the actual LaTeX command that we need to use... Essentially because now we have more than one copyright holder, we need to reference the section where the list is specified.